spec/observables/fromEvent-spec.ts

```diff
@@ -432,4 +432,34 @@ describe('fromEvent', () => {
     expect(nodeList[0]._removeEventListenerArgs).to.deep.equal(nodeList[0]._addEventListenerArgs);
     expect(nodeList[1]._removeEventListenerArgs).to.deep.equal(nodeList[1]._addEventListenerArgs);
   });
+
+  /**
+   * Huan(202111): Correct typing inference for Node.js EventEmitter as `number`
+   *  @see https://github.com/ReactiveX/rxjs/pull/6669
+   */
+  it('should successful inference the first argument from the listener of Node.js EventEmitter', (done) => {
```

Comment:
These sorts of tests we do with `dtslint`. They're under a different directory and are run with `npm run dtslint`.

Reply:
Moved to `dtslint`